packages/panda-san/src/navbar/navbar.scss

```diff
@@ -0,0 +1,82 @@
+.navbar {
+  align-items: center;
+  background-color: $navbar-color;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  left: 0;
+  position: relative;
+  width: 100%;
+  z-index: $stack-sticky;
+
+  &__brand {
+    display: flex;
+    flex: 1;
+  }
+
+  &__burger {
+    display: flex;
+    flex: 1;
+    justify-content: flex-end;
+  }
+
+  &__menu {
+    display: none;
+
+    &--active {
+      background-color: $navbar-menu-color;
+      display: flex;
+      flex-direction: column;
+      position: relative;
+      width: 100%;
+    }
+  }
+
+  &__item {
+    align-items: center;
+    display: flex;
+    height: px-to-rem(36px);
+    justify-content: flex-start;
+    width: 100%;
+
+    &:hover,
+    &:focus {
+      background-color: darken($navbar-menu-color, 10%);
+    }
+  }
+
+  @each $breakpoint-name, $gutter in $container-gutters {
+    @include breakpoint($breakpoint-name) {
+      padding: $gutter $gutter;
+    }
+  }
+
+  @include breakpoint(tablet) { /* stylelint-disable-line order/order */
```

Comment:
We should probably update this rule in our stylelint as this is a pattern we will be using everywhere IMO.